pyproject.toml

```diff
@@ -9,13 +9,20 @@ requires = [
 
 [tool.towncrier]
     # Do no set this since it is hard to import numpy inside the source directory
-    # Use "--version Numpy" instead
-    #project = "numpy"
-    filename = "doc/release/latest-note.rst"
-    directory = "doc/release/upcoming_changes"
+    # the name is hardcoded. Use "--version 1.18.0" to set the version
+    single_file = false
+    filename = "doc/release/{version}-notes.rst"
+    directory = "doc/release/upcoming_changes/"
     issue_format = "`gh-{issue} <https://github.com/numpy/numpy/pull/{issue}>`__"
     template = "doc/release/upcoming_changes/template.rst"
-    underlines="~="
+    underlines = "~="
+    wrap = false
+    all_bullets = false
+
+    [[tool.towncrier.type]]
+        directory = "highlight"
+        name = "Highlights"
+        showcontent = true
```

Comment:
Is this section repeated?

Reply:
opst, yes thanks. Adding a test that all fragments are picked up, does not check PR numbers, but other spelling errors. (have to try it though)

Reply:
Ohh, your PR with the restructure moved paths around. Should we keep the `upcoming_release` folder where it is?

Reply:
Now that you ask, I am not sure. Putting it in `doc/source/release` means it gets lost in the noise there. I think it is OK where it is.